src/components/pagination/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { FC, PropsWithChildren, useMemo } from 'react';
+import { useMultiStyleConfig } from '@chakra-ui/react';
+
+import PaginationButton from './PaginationButton';
+import { ChevronLeftSmallIcon, ChevronRightSmallIcon } from '../icons';
+import Box from '../box';
+
+const Dots = '...';
+const range = (start: number, end: number): Array<number> => {
+  const length = end - start + 1;
+  return Array.from({ length }, (_, idx) => idx + start);
+};
+
+export interface Props {
+  count: number;
```

Comment:
can we go with something like `currentPage`, `totalPages` ? I have a hard time knowing how to use the interface. What exactly is siblingCount doing?

Reply:
the interface is almost the same as on https://mui.com/material-ui/react-pagination/#basic-pagination
but yea `currentPage` and `totalPages` might be more clear to use.

`siblingCount` this is a count of left and right siblings around current page